tools/topology/topology2/development/CMakeLists.txt

```diff
@@ -13,7 +13,9 @@ set(TPLGS
 
 # CAVS SSP topology for TGL
 "cavs-nocodec\;sof-tgl-nocodec\;NUM_DMICS=4,PDM1_MIC_A_ENABLE=1,PDM1_MIC_B_ENABLE=1,\
-PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-sof-tgl-nocodec.bin,DEEPBUFFER_FW_DMA_MS=100"
+PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-sof-tgl-nocodec.bin,DEEPBUFFER_FW_DMA_MS=100,\
+SSP0_MIXER_2LEVEL=1"
```

Comment:
@lyakh this is good but why not MTL too?

Reply:
@ranj063 because we don't test MTL in PR CI yet. Maybe we can enable MTL separately after this one is merged and has run a daily test